composeApp/src/androidMain/kotlin/sp/bvantur/tasky/core/di/PlatformModule.android.kt

```diff
@@ -17,4 +23,15 @@ actual val platformModule = module {
     singleOf(::SecurePersistentStorageProviderImpl).bind<SecurePersistentStorageProvider>()
 
     single<TaskyDatabase> { createRoomDatabase(get(), get()) }
+
+    single {
+        CoroutineScope(SupervisorJob())
+    }
```

Comment:
It is not clear what and why you are providing this coroutine scope. By defining this inside your application class and giving it a fitting name, you can communicate here that you are providing an application-wide coroutine scope.